llvm/lib/Target/EraVM/EraVMOpcodes.td

```diff
@@ -154,61 +154,64 @@ def OpShl  : EraVMOpcode<"shl",  463, ArithNCommEncoding>;
-def OpDelegate : EraVMOpcode<"far_call.delegate", 1061, FarCallEncoding>; // is_shard is_static ⇒ 1061 + 2 × is_static + is_shard
-def OpMimic    : EraVMOpcode<"far_call.mimic",    1065, FarCallEncoding>; // is_shard is_static ⇒ 1065 + 2 × is_static + is_shard
-
-def OpRet           : EraVMOpcode<"ret.ok",              1069, DirectEncoding>; // to_label ⇒ 1069 + to_label
-def OpRetToLabel    : EraVMOpcode<"ret.ok.to_label",     1070, DirectEncoding>;
-def OpRevert        : EraVMOpcode<"ret.revert",          1071, DirectEncoding>; // to_label ⇒ 1071 + to_label
-def OpRevertToLabel : EraVMOpcode<"ret.revert.to_label", 1072, DirectEncoding>;
-def OpPanic         : EraVMOpcode<"ret.panic",           1073, DirectEncoding>; // to_label ⇒ 1073 + to_label
-def OpPanicToLabel  : EraVMOpcode<"ret.panic.to_label",  1074, DirectEncoding>;
-
-def OpLoadHeap        : EraVMOpcode<"ld.1",     1075, HeapOpEncoding>; // src inc ⇒ 1075 + 10 × src + inc
-def OpLoadHeapInc     : EraVMOpcode<"ld.1.inc", 1076, HeapOpEncoding>;
-def OpStoreHeap       : EraVMOpcode<"st.1",     1077, HeapOpEncoding>; // src inc ⇒ 1077 + 10 × src + inc
-def OpStoreHeapInc    : EraVMOpcode<"st.1.inc", 1078, HeapOpEncoding>;
-
-def OpLoadAuxHeap     : EraVMOpcode<"ld.2",     1079, HeapOpEncoding>; // src inc ⇒ 1079 + 10 × src + inc
-def OpLoadAuxHeapInc  : EraVMOpcode<"ld.2.inc", 1080, HeapOpEncoding>;
-def OpStoreAuxHeap    : EraVMOpcode<"st.2",     1081, HeapOpEncoding>; // src inc ⇒ 1081 + 10 × src + inc
-def OpStoreAuxHeapInc : EraVMOpcode<"st.2.inc", 1082, HeapOpEncoding>;
+
+def OpPtrAdd    : EraVMOpcode<"addp",  847, ArithPtrEncoding>;
+def OpPtrSub    : EraVMOpcode<"subp",  895, ArithPtrEncoding>;
+def OpPtrPack   : EraVMOpcode<"pack",  943, ArithPtrEncoding>;
+def OpPtrShrink : EraVMOpcode<"shrnk", 991, ArithPtrEncoding>;
+def OpCall      : EraVMOpcode<"call",  1039, DirectEncoding>;
+
+def OpContextThis              : EraVMOpcode<"this", 1040, DirectEncoding>;
+def OpContextCaller            : EraVMOpcode<"par",  1041, DirectEncoding>;
+def OpContextCodeAddress       : EraVMOpcode<"code", 1042, DirectEncoding>;
+def OpContextMeta              : EraVMOpcode<"meta", 1043, DirectEncoding>;
+def OpContextErgsLeft          : EraVMOpcode<"ergs", 1044, DirectEncoding>;
+def OpContextSp                : EraVMOpcode<"sp",   1045, DirectEncoding>;
+def OpContextGetContextU128    : EraVMOpcode<"ldvl", 1046, DirectEncoding>;
+def OpContextSetContextU128    : EraVMOpcode<"stvl", 1047, DirectEncoding>;
+def OpContextIncrementTxNumber : EraVMOpcode<"inctx", 1049, DirectEncoding>;
+
+// Removed in VM 1.5.0
+def OpContextSetErgsPerPubdataByte : EraVMOpcode<"stpub", 1048, DirectEncoding>;
+
+def OpSload     : EraVMOpcode<"lds", 1050, DirectEncoding>;
+def OpSstore    : EraVMOpcode<"sts", 1051, DirectEncoding>;
+
+def OpLogToL1       : EraVMOpcode<"logl1",   1052, DirectEncoding>; // is_first ⇒ 1052 + is_first
+def OpLogToL1First  : EraVMOpcode<"logl1.f", 1053, DirectEncoding>;
```

Comment:
Let's change `f` => `i`, `is_first` => `is_initializer`.
`i` (initializer) better reflects the purpose of this instruction.

Reply:
Updated here and `log.f` below.